### PR TITLE
Registration Administration V2 QA Changes

### PR DESCRIPTION
**Pull request by FinnIckler**
Things from the sheet that I haven't done:
- ~~Make EventIcons Smaller (currently we don't have a size Parameter for EventIcon)~~ :heavy_check_mark: 
- ~~Fix alignment issues for empty tables (not sure how to fix...)~~ :heavy_check_mark:

### CODE REVIEW

**Comment by dunkOnIT** [comment]
@thewca-bot deploy staging

File: app/views/registrations/edit_v2.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<% provide(:title, t('registrations.edit_registration.title', person: @user.person.name, comp: @competition.name)) %>
+<% provide(:title, t('registrations.edit_registration.title', person: @user.person&.name || @user.id, comp: @competition.name)) %>
```
Comment (gregorbg) [inline app/views/registrations/edit_v2.html.erb:1]: I think you can use `@user.name` here instead of `@user.id`

**Review by gregorbg — APPROVED** [review]

File: app/views/registrations/edit_v2.html.erb (hunk 1 above)
Comment (FinnIckler) [reply app/views/registrations/edit_v2.html.erb]: ah okay makes sense

**Comment by dunkOnIT** [comment]
@thewca-bot deploy staging

**Comment by dunkOnIT** [comment]
@thewca-bot deploy staging

**Comment by dunkOnIT** [comment]
@thewca-bot deploy staging

**Comment by dunkOnIT** [comment]
@thewca-bot deploy staging

**Comment by dunkOnIT** [comment]
@thewca-bot deploy staging

**Comment by dunkOnIT** [comment]
Gregor, will you want to re-review? Several changes have been made after you approved.

**Review by gregorbg — APPROVED** [review]

**Pull request merged**